website/docs/reference/widgets/filepicker.md

```diff
@@ -110,19 +90,18 @@ Properties allow you to customize the widget, connect it to other widgets and tr
-| **Maximum No. of files**              | Sets the maximum number of files allowed to be uploaded by a user.                                                                                                                                                                                                                                                                 |
-| **Maximum File Size**              | Sets the maximum allowed size of each file that a user can upload.                                                                                                                                                                                                                                |
-| **Allowed File Types**                | Controls which types of files a user is allowed to upload. Accepts an array of wildcards image/, exact mime types image/jpeg, or file extensions e.g.: '.jpg', '.jpeg', '.png', '.gif']. The following file types are supported: Images, Videos, Audio, Text, MS Word, JPEG, and PNG.                          |
-| **Data Format**                   | Determines the data format of the files uploaded. You can choose from Base64, Binary, , Text, and Array(CSV).                                                                                                                                                                                                                      |
-| **Infer data-types from CSV**       | Enables or disables the automatic inference of data types from CSV files.                                                                                      |
-| **Required**                           | Makes input to the widget mandatory.                                                                                                            |
-| **Visible**             | Controls widget's visibility on the page.                                                                           | `{{FilePicker.isVisible}}`  |
-| **Disabled**                 | Makes the widget un-clickable or unusable. The widget remains visible to the user but user interaction won't be allowed.                                                                                                                                                                           | `{{FilePicker.isDisabled}}` |
-| **Animate Loading**               | Allows you to control a widget’s animation on the page load.                                                                                                   |                             |
-
+| Property                      	| Data Type       	| Description                                                                                                                                                                                                                                                                           	|
```

Comment:
no need of code blocks for datatypes. Also add data type column for reference and style properties